### DISCUSSION TITLE
Feature Request: Support syncing the queue

### DISCUSSION THREAD

**Discussion by LivInTheLookingGlass** [thrillfall/nextcloud-gpodder#27] [Ideas]
It would be really nice if synchronization here would also synchronize the queue between devices. I'm not sure that that's actually in the scope of this project, or the AntennaPod sync service, but it would be super useful

**Comment by thrillfall** [comment]
Its currently not in scope but I thought about this too.
Once we have proof this app integrates at least with AntennaPod I'd definitely go for it. 
Be aware that syncing queue is not implemented in AntennaPod yet. Syncing subscriptions and episode states was already implemented

**Comment by LivInTheLookingGlass** [comment]
That sounds pretty reasonable

Thank you so much for making this, by the way. It's definitely going to make my life a little easier. The actual gpodder service was quite unreliable when I tried it

**Comment by agates** [comment]
So, it's been a few years.  Would it be reasonable to revisit this?

EDIT: For anyone else interested https://github.com/orgs/OpenPodcastAPI/discussions/7

**Reply by thrillfall** [reply]
absolutely. feel very welcome to start development
